### ISSUE TITLE
Cocogen allows nodeset as attribute

### ISSUE THREAD

**Issue by MitchellWeg** [CoCoNut-UvA/coconut#13]
During compilation, I get an error stating `ast.h` wasn't found when using Nodesets. This isn't weird, considering a `Nodeset` relies on the fact that it's members are already defined. But what I think is weird, is that cocogen happily goes along and generates the build.

Add this snippet to a clean cloned civicc-skeleton:

```
nodeset Foo = Bar | Baz

node MyNode {
   attributes {
       Foo f
  }
}
```

Then run: `./configure`

**Comment by TristanLaan** [comment]
Thanks for reporting this! This should be fixed in 689df503eb19209be7e76b24d96524546e99e861. It seems to be a compound issue of:
1. Cocogen not properly checking if nodeset members actually exist. This caused a segfault later on.
2. Cmake not properly checking the return status of cocogen and "eating" the line that said "Segmentation fault". Leaving the user with only a status message "-- Generating with cocogen" and no way to tell that cocogen crashed in the background.

As far as I can tell this is not actually related to setting a nodeset as attribute. This is perfectly allowed, and the code snippet compiles as long as Bar and Baz are defined.

**Comment by TristanLaan** [comment]
The update has just been merged, so I'll close this issue for now. Feel free to reopen it if you are still having problems.
